guide/multiple-wallets.md

```diff
@@ -0,0 +1,313 @@
+
+{% include picture.html
+   image = "/assets/images/guide/multiple-wallets/settings.png"
+   retina = "/assets/images/guide/multiple-wallets/settings@2x.png"
+   modalImage = "/assets/images/guide/multiple-wallets/settings-big.png"
+   alt-text = "A settings screen with a list of wallets at the top"
+   caption = "Application and wallet-specific settings are clearly separated."
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+%}
+
+[Settings]({{ '/guide/daily-spending-wallet/settings/' | relative_url }}) are organized into application-wide settings and wallet-specific settings.
+
+A user may choose different security options across wallets, for example, to reduce friction for small lightning payments, and better security for wallets that store larger amounts.
+
+</div>
+
+## Archival
+
+Based on the principle of [progressive security]({{ '/guide/getting-started/principles/#security' | relative_url }}), the [upgradeable wallet reference design]({{ '/guide/upgradeable-wallet/' | relative_url }}) presents an application in which users can move from one security model to another.
+
+An example is a switch from a single-key to a multi-key wallet, because the users funds have increased in value. By necessity, this requires a new wallet to be created and funds moved over. This, and other scenarios, may result in users having multiple older wallets that they no longer actively use. But, they may still want to keep an eye on those wallets in case an unexpected payment arrives.
+
+This introduces the need for a wallet archive. Wallets in this archive are tucked away and generally not accessible in the primary user flows. However, the application still looks for incoming transactions and notifies the user accordingly.
+
+{% include image-gallery.html pages = page.images_archive %}
+
+## Alternate approaches
+
+#### Card-based wallet visualization
+
+A card format sometimes chosen to give wallets a unique look that helps users recognize what it is they are looking at. Some applications also let users customize the appearance, which helps with recognition even more. Regardless of that, it is also recommended to communicate primary properties, like the layer and wallet type (single-key, multi-key, view-only…).
+
+{% include image-gallery.html pages = page.images_visuals %}
+
+#### On wallet accounts
+
+Different applications use the terms wallet and account to refer to different things (see the [glossary wallet page]({{ '/guide/glossary/wallet/' | relative_url }})). A specific meaning to be aware of, and decide whether to support, is that of account path levels in hierarchical deterministic wallets (see [BIP 44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#user-content-Account)).
+
+To put it simply, multiple separate wallets can be derived from the same recovery phrase, by choosing a different account number. This feature may be helpful to explicitly separate funds meant for different purposes.
```

Comment:
```suggestion
To put it simply, multiple separate wallets can be unlocked using the same recovery phrase, simply by choosing a different account number. This feature may be helpful to explicitly separate funds meant for different purposes.
```

Reply:
Unlocked is not quite the same as derived.